src/object/obj_ec.h

```diff
@@ -793,7 +793,7 @@ obj_ec_parity_lists_match(struct daos_recx_ep_list *lists_1,
 			    (list_1->re_items[j].re_recx.rx_nr !=
 			     list_2->re_items[j].re_recx.rx_nr)) {
 				D_ERROR("got different parity recx in EC data recovery\n");
-				return -DER_IO;
+				return -DER_DATA_LOSS;
```

Comment:
just confirm that is it must needed to change to DATA_LOSS err code? Seems EIO also work?

Reply:
yes, DER_IO is too general, and also cause rebuild to fail, while data_loss will tell us more information and also rebuild will skip the container, instead of failing.